pkg/service/store.go

```diff
@@ -67,6 +67,11 @@ func (s *ClusterService) String() string {
 	return s.Cluster + "/" + s.Namespace + ":" + s.Name
 }
 
+// NamespaceServiceName returns the namespace and service name
+func (s *ClusterService) NamespaceServiceName() string {
+	return s.Namespace + "/" + s.Name
```

Comment:
Seems odd in the `String()` above the Namespace and Name are split with `:` and here with `/`